interface/src/app/gene/search/search.js

```diff
@@ -264,18 +264,21 @@ angular.module('adage.gene.search', [
           for (var gene in $scope.selectedGenes) {
```

Comment:
This looks like a for loop over elements of an array. This seems like the case where we want to use the traditional loop over elements to avoid snagging a property.
http://stackoverflow.com/questions/1963102/what-does-the-jslint-error-body-of-a-for-in-should-be-wrapped-in-an-if-statemen